tests/test_base_script.py

```diff
@@ -248,11 +268,21 @@ def test_non_configurable_script_get_schema(self):
                 await script.do_configure(configure_data)
             self.assertEqual(script.state.state, ScriptState.UNCONFIGURED)
 
-            # now real configuration
+            # now real configuration; specify nonexistent checkpoints
+            # just to test that the configure command handles checkpoints
+            # also specify a log level to see if that works
             wait_time = 0.5
-            await self.configure_script(script, wait_time=wait_time)
-
-            # set a pause checkpoint
+            # specify a log level that is not the default (which is INFO)
+            # and is only slightly more verbse than INFO
+            log_level = logging.INFO - 1
+            prelim_pause_checkpoint = "preliminary nonexistent pause checkpoint"
+            prelim_stop_checkpoint = "preliminary nonexistent stop checkpoint"
+            await self.configure_script(script, wait_time=wait_time,
+                                        log_level=log_level,
+                                        pause_checkpoint=prelim_pause_checkpoint,
+                                        stop_checkpoint=prelim_stop_checkpoint)
```

Comment:
Should this above block have an assert? If the goal is to check that `configure_script` doesn't raise itself, I like making that explicit by wrapping it in:

```
try:
    do_thing()
except BadException:
    self.fail("shouldn't have raised BadException")
```

Reply:
That is exactly why I documented `configure_script` to raise AssertionError if the configuration failed.

Reply:
I thought that tests that `error` out (instead of `fail`ing) don't `assert`, but rather give whatever exception was raised by the called function?

Reply:
This was due to a confusion on my part. No try:except assert block necessary here.